### PR TITLE
Begin spec testing

### PR DESCRIPTION
**Pull request by xaque208**
This just ensures that the catalog compiles on all supported platforms
and lays the groundwork for future rspec-puppet.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,7 +2,6 @@
 Gemfile.lock
 .*.sw?
 pkg
-spec/fixtures
```
Comment (igalic) [inline .gitignore]: why remove this from gitignore?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+require 'rspec-puppet'
```
Comment (igalic) [inline spec/spec_helper.rb]: oh, you're not using puppetlabs-rspec-helper here?

**Comment by xaque208** [comment]
@igalic I just wanted something basic for the time being.  Feel free to make improvements where warranted.

File: .gitignore (hunk 1 above)
Comment (xaque208) [inline .gitignore]: I actually think I removed that only for temporary addition of the modules link.  That can likely be added back in.

**Comment by igalic** [comment]
*nod*

**Pull request merged**